release/models/acl/openconfig-packet-match.yang

```diff
@@ -118,6 +118,17 @@ module openconfig-packet-match {
       description
         "Ethertype field to match in Ethernet packets";
     }
+
+    leaf dot1p {
```

Comment:
This was taken directly from https://github.com/openconfig/public/blob/9152b712c79d0fcd776ec4b8efc9f655f6c5b916/release/models/qos/openconfig-qos-elements.yang#L1311. I was unsure of how to reuse the other definition or whether I _should_, so I chose to just copy it. Is this acceptable, or is there a way to reduce repetition?